main.go

```diff
@@ -90,6 +103,15 @@ It can be used in a curl-like fashion when benchmarking features are disabled.
 		return opts, errExit
 	}
 
+	// Read defaults from ~/.yab.ini
+	if user, err := user.Current(); err != nil {
```

Comment:
If we can find a xdg package that works well across all the OSes, then sure. I think we want:
~/.config/yab for OSX and Linux
%APPDATA% for Windows

You can see what the most imported packages for Go are using godoc:
https://godoc.org/?q=xdg

We'll want to find that that's used in other packages but follows conventions in other operating systems too.